docs/DeveloperGuide.md

```diff
@@ -270,14 +270,17 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​                                | I want to …​                              | So that I can…​                                             |
+|----------|----------------------------------------|-------------------------------------------|-------------------------------------------------------------|
+| `* * *`  | user                                   | have a list feature                       | list all the recipes that I have sorted                     |
+| `* * *`  | user                                   | have a load and save feature              | load and save my recipes automatically                      |
+| `* * *`  | user                                   | have a view command feature               | view my recipes                                             |
```

Comment:
think we can change from "view command feature" to just "view feature"? and I think we can remove the "so I can view my recipes" part since I think it's quite self explanatory(?)